tests/test_timeline_ui.py

```
@@ -0,0 +1,298 @@
+
+    @pytest.fixture
+    def timeline_ui(self):
+        """Create TimelineUI instance for testing."""
+        return TimelineUI()
+
+    @pytest.fixture
+    def sample_timeline(self):
+        """Create sample timeline for testing."""
+        base_time = datetime.now()
+        events = []
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)